src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1663,9 +1677,25 @@ namespace Sched {
                     }
                 }
 
+                // curcolCount is 1-indexed here
+                if (static_cast<size_t>(curcolCount) > result->second["values"].size()) {
+                    ShowSevereCustom(
+                        state, eoh, format("Requested column number {}, but found only {} columns.", curcolCount, result->second["values"].size()));
+                    ShowContinueError(state, fmt::format("Error Occurred in {}", state.files.TempFullFilePath.filePath));
+                    ShowFatalError(state, "Program terminates due to previous condition.");
+                }
```

Comment:
Schedule:File: handle the case where I might request column 10000 from a CSV file that actually has 2 columns.

Otherwise you get a cryptic json error